test/functional/apps/management/_field_formatter.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export default function ({ getService, getPageObjects }) {
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+  const browser = getService('browser');
+  const PageObjects = getPageObjects(['settings']);
+  const testSubjects = getService('testSubjects');
+
+  describe('field formatter', function () {
+    this.tags(['skipFirefox']);
+
+    before(async function () {
+      await browser.setWindowSize(1200, 800);
+      await esArchiver.load('discover');
+      // delete .kibana index and then wait for Kibana to re-create it
```

Comment:
nit: there is a method `await esArchiver.emptyKibanaIndex();`. I wonder if using it would be a "right" way to clear `.kibana`

(not 100% sure if those are equal)

Reply:
I think that comment wasn't accurate, but I did employ `await esArchiver.emptyKibanaIndex();` in the `after` function